eos/statistics/log-likelihood.hh

```diff
@@ -290,14 +290,15 @@ namespace eos
             }
 
             /*!
-             * Create a new LogLikelihoodBlock for a unform bound on the parameter space expressed through a single observable.
+             * Create a new LogLikelihoodBlock for a uniform bound by summing individual contributions and applying a gaussian-like penalty
+             * to events exceeding the bound.
```

Comment:
Event vs parameter space. Do you have a good argument why the new/former is correct?

Reply:
I don't, I just wanted to mention the penalty function and I couldn't construct a nice sentence with `parameter space`. Basically I would need to say that we restrict some region of the parameter space with a Gaussian like penalty, but I thought that mentioning events would make the sentence simpler to understand.

Reply:
How about you just replace ``to events exceeding the bound`` with ``beyond the bound``?

Reply:
Ok!